applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py

```diff
@@ -262,6 +271,13 @@ def __init__(self, model, custom_settings):
         ## Construct the linear solver
         self.linear_solver = linear_solver_factory.ConstructSolver(self.settings["linear_solver_settings"])
 
+        if not self.settings["turbulence_model"].IsEquivalentTo(KratosMultiphysics.Parameters("{}")):
+            self.turbulence_model_solver = CreateTurbulenceModel(model, self.settings["turbulence_model"])
+            self.condition_name = self.turbulence_model_solver.GetFluidVelocityPressureConditionName()
+            KratosMultiphysics.Logger.PrintInfo("NavierStokesSolverMonolithic", "Using " + self.condition_name)
```

Comment:
can you make this a bit more specific?
It is related to the turbulence model right?

Reply:
yes, i made the naming more explanatory :)